src/extension/service.ts

```diff
@@ -87,6 +88,13 @@ Object.assign(globalThis, {
     getCurrentNetworkWorkerService,
     ECKeyIdentifier,
     IdentifierMap,
+    BigNumber,
+})
+Object.defineProperty(BigNumber.prototype, '__debug__amount__', {
```

Comment:
the is global hacking. move to other file?

Reply:
I think it's ok to do this